lambda/executor.js

```diff
@@ -116,14 +127,17 @@ const runInParallel = async(num, lambdaARN, lambdaAlias, payloads, preARN, postA
         if (invocationResults.FunctionError) {
             throw new Error(`Invocation error (running in parallel): ${invocationResults.Payload} with payload ${JSON.stringify(actualPayload)}`);
         }
+        if(!isNan(sleepBetweenRunsMs) && sleepBetweenRunsMs > 0) {
```

Comment:
Added here and line 149 as discussed in the Issue, but I was on the fence if it's better suited in the util library where the actual invocation takes place to reduce duplicated code. Here may be fine though

Reply:
I think it makes sense here - I already made the "error" of passing down way too many parameters to the util functions :)